Apps/Contoso.Forms.Puppet/Contoso.Forms.Puppet.Droid/Contoso.Forms.Puppet.Droid.csproj

```diff
@@ -19,6 +19,7 @@
     <AndroidUseLatestPlatformSdk>true</AndroidUseLatestPlatformSdk>
     <AndroidTlsProvider>
     </AndroidTlsProvider>
+    <AndroidEnableMultiDex>true</AndroidEnableMultiDex>
```

Comment:
Wow, we are just a hello world, that would mean every single customer would hit this? This needs further investigation.